src/redux/books/books.js

```diff
@@ -0,0 +1,45 @@
+function makeIdWithLength(length) {
+  let result = '';
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  const charactersLength = characters.length;
+
+  for (let i = 0; i < length; i += 1) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
+
+const uuid = makeIdWithLength(20);
+// console.log(uuid);
+
+//
+const ADD_BOOK = 'ADD_BOOK';
+const DELETE_BOOK = 'DELETE_BOOK';
```

Comment:
- [ ] Please let's make sure to always scope action types based on the feature they represent. This is very important especially if you have plans to scale your app in the future. You will be required to do something like this: 
 
  `const ADD_BOOK = 'bookstore/books/ADD_BOOK';`
   
  This is referred to as the [ducks pattern](https://github.com/erikras/ducks-modular-redux)